doc/decisions/0002-sign-commits-with-gpg.md

```diff
@@ -0,0 +1,44 @@
+# 2. Sign Commits With GPG
+
+| Date       | Tags              |
+|------------|-------------------|
+| 2018-02-16 | process, security |
+
+
+## Status
+
+Proposed
+
+## Context
+
+OpenBounty is a system which has value flowing through it.
+Naturally security is a concern that should be taken into consideration.
+
+Currently an attacker might get access to an account of a team member
+and pose as that developer, merging PRs and pushing changes.
+
+## Decision
+
+In order to verify that commits in the repository are actually authored by the specified
+author we adopt [GPG signing of Git commits](https://git-scm.com/book/id/v2/Git-Tools-Signing-Your-Work).
+
+This will allow us to verify authenticity of the author information saved in
+a Git commit and make workflows like deploying on push safer.
+
+It also introduces some complexity because contributors who want to sign
```

Comment:
@pablodip I think it's part of the configuration of this Bot, not GitHub itself: https://github.com/jarrodldavis/probot-gpg

#296 is an example where the GPG check has been skipped because the contributor was not an org member.

Reply:
Okay, got it. Probably worth mentioning it.